Dangerfile

```diff
@@ -1,119 +1,220 @@
+    ].freeze
+    PR_SIZE = { recommended: 200, maximum: 500 }.freeze
+
+    def run
+      return error if lines_changed > PR_SIZE[:maximum]
+      return warning if lines_changed > PR_SIZE[:recommended]
+
+      info
+    end
+
+    private
+
+    def error
+      { severity: :error, message: error_message }
+    end
+
+    def error_message
+      <<~EMSG
+        This PR changes `#{lines_changed}` LoC (not counting whitespace/newlines).
+
+        In order to ensure each PR receives the proper attention it deserves, those exceeding
+        `#{PR_SIZE[:maximum]}` will not be reviewed, nor will they be allowed to merge. Please break this PR up into
+        smaller ones.
+
+        If you have reason to believe that this PR should be granted an exception, please see the
+        [Code Review Guidelines FAQ](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/code_review_guidelines.md#faq).
+
+        #{file_summary}
 
-  exclusion_text = <<~EXCLUSIONS_TEXT
+        Big PRs are difficult to review, often become stale, and cause delays.
+      EMSG
+    end
 
-  #### Exclusions
+    def warning
+      { severity: :warning, message: warning_message }
+    end
 
-  - #{excluded_files.collect { |key, val| "#{key} (+#{val[:insertions]}/-#{val[:deletions]} )" }.join("\n- ")}
+    def warning_message
+      <<~EMSG
+        This PR changes `#{lines_changed}` LoC (not counting whitespace/newlines).
 
-  ####
+        In order to ensure each PR receives the proper attention it deserves, we recommend not exceeding
+        `#{PR_SIZE[:recommended]}`. Expect some delays getting reviews.
 
-  _Note: We exclude the following files when considering PR size_
+        #{file_summary}
 
-  ```
-  #{EXCLUSIONS}
+        Big PRs are difficult to review, often become stale, and cause delays.
+      EMSG
+    end
 
-  ```
+    def info
+      { severity: :info, message: 'All set.' }
+    end
 
-  EXCLUSIONS_TEXT
+    def file_summary
+      <<~MSG
+        <details>
+          <summary>File Summary</summary>
 
+          #### Files
+          #{changes.collect { |change| "- #{change.file_name} (+#{change.insertions}/-#{change.deletions})" }.join "\n"}
 
-  file_summary = <<~HTML
-    <details><summary>File Summary</summary>
+          ####
+          _Note: We exclude files matching the following when considering PR size:_
 
-    #### Included Files
+          ```
+          #{EXCLUSIONS.join ', '}
+          ```
+        </details>
+      MSG
+    end
 
-    - #{included_files.collect { |key, val| "#{key} (+#{val[:insertions]}/-#{val[:deletions]} )" }.join("\n- ")}
-    #{exclusion_text if excluded_files.any?}
-    </details>
-  HTML
+    def lines_changed
+      @lines_changed ||= changes.sum(&:total_changes)
+    end
 
-  footer = 'Big PRs are difficult to review, often become stale, and cause delays.'
+    def changes
+      `#{files_command}`.split("\n").map do |line|
```

Comment:
```suggestion
      @changes ||= `#{files_command}`.split("\n").map do |line|
```
changes is used by lines_changed and file_summary so we shouldn't shell out twice